packages/hooks/src/hooks.spec.tsx

```diff
@@ -269,6 +271,7 @@ describe('packages/hooks', () => {
       act(() => {
         document.dispatchEvent(new Event('visibilitychange'));
       });
+      jest.advanceTimersByTime(30e3);
```

Comment:
Should this be replaced by the new `waitForState` util?

Reply:
Possibly, though I'm not too familiar with this hook, so I used the same logic as on L264

Reply:
I don't think `waitForState` can replace `advanceTimersByTime`.  We ran into issues with some DatePicker tests where awaiting an act call didn't wait for some popover callbacks to finish. We had to use `advanceTimersByTime`. I could be wrong though.